test/system/012-manifest.bats

```diff
@@ -56,4 +89,46 @@ load helpers.registry
     is "$output" ".*\"mediaType\": \"application/vnd.docker.distribution.manifest.list.v2+json\"" "Verify --tls-verify=false with REGISTRY_AUTH_FILE works against an insecure registry"
 }
 
+@test "manifest list --add-compression with zstd" {
+    if ! type -p skopeo; then
+        skip "skopeo not available"
+    fi
```

Comment:
to my knowledge skopeo is a hard requirement so this skip makes no sense
cc @edsantiago

Reply:
The tests which are using `skopeo` have this check directly or indirectly so I have added it here.